docs/assets/includes/self-hosted-version.txt

```diff
@@ -0,0 +1,4 @@
+!!! tip
+    To see the version of your Codacy Self-hosted instance click your avatar on the top right-hand corner of any Codacy page: 
+
+    ![Obtaining the Codacy Self-hosted version](./images/self-hosted-version.png)
```

Comment:
I decided to include the instructions on how to see the version in a "tip" admonition card in a centralized file that is then included in the pages where this information is useful.